.rubocop.yml

```diff
@@ -39,6 +39,12 @@ Metrics/BlockLength:
     - 'app/services/forms_grouper.rb'
     - 'app/services/notes_grouper.rb'
 
+Metrics/ClassLength:
+  Exclude:
```

Comment:
I'm not sure we should move these here, because this to me indicates that we never want to fix them (and we don't care how big they grow). I don't think that is the intent here. If I'm wrong about that, please move this to a separate PR and we can discuss there.

Reply:
We have different philosophies about this, but not worth the effort to litigate. I backed this out, but re-running autogen resulted in a bunch of other changes.